packages/applications/ssr/src/components/pages/attestation-conformité/FormulaireAttestationConformité.tsx

```diff
@@ -109,6 +105,20 @@ export const FormulaireAttestationConformité: FC<FormulaireAttestationConformit
 
         {demanderMainlevée.visible && (
           <>
+            <Checkbox
+              id="demanderMainlevee"
+              state={validationErrors.includes('demanderMainlevee') ? 'error' : 'default'}
+              options={[
+                {
+                  label: `Je souhaite demander une mainlevée de mes garanties financières`,
+                  nativeInputProps: {
+                    disabled: !demanderMainlevée.canBeDone,
```

Comment:
penser à rajouter un aria-disabled ici


```suggestion
                    disabled: !demanderMainlevée.canBeDone,
                    "aria-disabled": !demanderMainlevée.canBeDone,
```